cmd/create_onchain_transaction.go

```diff
@@ -55,11 +55,11 @@ var createOnchainTransactionCmd = &cobra.Command{
 			Rpc: &model.OnchainRpc{
 				Url:           utils.GetFlagStringValue(cmd, utils.UrlFlag),
 				SkipBroadcast: utils.GetFlagBoolValue(cmd, utils.SkipBroadcastFlag),
-				EvmParams: &model.OnchainEvmParams{
-					DisableDynamicGas:     utils.GetFlagBoolValue(cmd, utils.DisableDynamicGasFlag),
-					ReplacedTransactionId: utils.GetFlagStringValue(cmd, utils.ReplacedTransactionIdFlag),
-					ChainId:               utils.GetFlagStringValue(cmd, utils.ChainIdFlag),
-				},
+			},
+			EvmParams: &model.OnchainEvmParams{
+				DisableDynamicGas:     utils.GetFlagBoolValue(cmd, utils.DisableDynamicGasFlag),
```

Comment:
while you are here. disable_dynamic_nonce was added as a boolean option

Reply:
It's a boolean in the request. What's wrong?

Reply:
Nm. I see what the problem is

Reply:
Nm. I see the problem.

Reply:
Moving to backlog https://github.com/coinbase-samples/prime-cli/issues/12